loginpage/index.html

```diff
@@ -70,29 +124,57 @@
           <input type = "checkbox" id = "remember">
           <input name = "type" type = "hidden" id = "remember_hidden" required>
           <label for = "remember" id = "rememberme">Remember me</label>
-          <a id = "signup" href="../signup/index.html" style="text-decoration:none"><p id = "signupopt">Sign Up</p></a>
+          <a id = "signup" href="../signup/index.html" style="text-decoration:none">
+            <p id = "signupopt">Sign Up</p>
+          </a>
         </div>
         <br><br>
         <input type="submit" id = "login" value="Login">
         <br><br>
-        <a id = "forgot-password" href="../forgotpassword/index.html" style="text-decoration:none"><p style="font-size:15px; margin-left:24%;">Forgot Password?</p></a>
+        <a id = "forgot-password" href="../forgotpassword/index.html" style="text-decoration:none">
+          <p style="font-size:15px; margin-left:24%;">Forgot Password?</p>
+        </a>
+=======
+    <div class="fields">
+     <input name="login" type="email" id="email" placeholder="Email" autofocus required>
+    </div>
+    <div class="bar">
+      <i></i>
+    </div>
+    <div class="fields">
+     <input name="password" id="pass" type="password" placeholder="Password" required>
+    </div>
+    <br>
+    <div class="options">
+     <input type="checkbox" id="remember">
+     <input name="type" type="hidden" id="remember_hidden" required>
+     <label for="remember" id="rememberme">Remember me</label>
+     <a id="signup" href="../signup/index.html" style="text-decoration:none">
+<p id="signupopt">Sign Up</p>
+</a>
+    </div>
+    <br><br>
+    <input type="submit" id="login" value="Login">
+    <br><br>
+    <a id="forgot-password" href="../forgotpassword/index.html" style="text-decoration:none"><p style="font-size:15px; margin-left:24%;">Forgot Password?</p></a>
```

Comment:
Line is longer than allowed. (161 > 129)

*LineLengthBear, severity NORMAL, section `default`.*